chapters/renderpass.txt

```diff
@@ -1802,22 +1802,10 @@ corresponding subpass dependency.
   * [[VUID-VkSubpassDependency2KHR-srcSubpass-03085]]
     pname:srcSubpass and pname:dstSubpass must: not both be equal to
     ename:VK_SUBPASS_EXTERNAL
-  * [[VUID-VkSubpassDependency2KHR-srcSubpass-03086]]
-    If pname:srcSubpass is equal to pname:dstSubpass, pname:srcStageMask and
-    pname:dstStageMask must: only contain one of
-    ename:VK_PIPELINE_STAGE_TOP_OF_PIPE_BIT,
-    ename:VK_PIPELINE_STAGE_DRAW_INDIRECT_BIT,
-    ename:VK_PIPELINE_STAGE_VERTEX_INPUT_BIT,
-    ename:VK_PIPELINE_STAGE_VERTEX_SHADER_BIT,
-    ename:VK_PIPELINE_STAGE_TESSELLATION_CONTROL_SHADER_BIT,
-    ename:VK_PIPELINE_STAGE_TESSELLATION_EVALUATION_SHADER_BIT,
-    ename:VK_PIPELINE_STAGE_GEOMETRY_SHADER_BIT,
-    ename:VK_PIPELINE_STAGE_FRAGMENT_SHADER_BIT,
-    ename:VK_PIPELINE_STAGE_EARLY_FRAGMENT_TESTS_BIT,
-    ename:VK_PIPELINE_STAGE_LATE_FRAGMENT_TESTS_BIT,
-    ename:VK_PIPELINE_STAGE_COLOR_ATTACHMENT_OUTPUT_BIT,
-    ename:VK_PIPELINE_STAGE_BOTTOM_OF_PIPE_BIT, or
-    ename:VK_PIPELINE_STAGE_ALL_GRAPHICS_BIT
+  * If pname:srcSubpass is equal to pname:dstSubpass, pname:srcStageMask and
+    pname:dstStageMask must: not set any bits that are not
```

Comment:
We'd rather use "neither ALL_GRAPHICSBIT, not one of" but otherwise this seems fine.

Reply:
done